packages/editor/src/components/convert-to-group-buttons/convert-button.js

```diff
@@ -61,7 +62,11 @@ export default compose( [
 
 		const groupingBlockName = getGroupingBlockName();
 
-		const groupingBlockAvailable = canInsertBlockType( groupingBlockName );
+		const rootClientId = clientIds && clientIds.length > 0 ?
+			getBlockRootClientId( clientIds[ 0 ] ) :
+			undefined;
+
+		const groupingBlockAvailable = canInsertBlockType( groupingBlockName, rootClientId );
```

Comment:
This doesn't work if you have a hierarchy like Cover > Columns > Group, as the root block (cover) can still be grouped. It seems like a general flaw of `canInsertBlockType`.

Can you think of any solution to that, @jorgefilipecosta? 

I still think this is an improvement that will cover a lot of cases, so if there's no easy fix it's still worth shipping this. In that case a separate issue can be created for improving `canInsertBlockType`.